content/news/ragls6-registrations.md

```diff
@@ -0,0 +1,13 @@
+<div style="text-align:center" markdown="1">
+![Red Alert Global League Season 6](/images/news/20181003-ragl-vi.png)
+</div>
+
+Attention, players!
+
+Red Alert Global League Season 6 is starting soon, with registrations closing on October 10th. The tournament itself is to be held from October 15th till December 23rd, ending with a Masters Finals to decide the next League champion.
+
+New to this season is the streamlining of the tier system, with Masters and Minions divisions set to be potentially increased (up to 16 players per group) and Recruits division removed in favour of pre-season qualification stage that will be used to rank league newcomers and fill vacant Minions division slots. Also, the main stage is getting packed with action now, as league rounds are double-stacked per week, and each player is to play two best-of-2 matches in his division each week.
```

Comment:
`in favour of pre-season qualification stage` &rarr; `in favour of a pre-season qualification stage`